src/tests/ftest/security/pool_connect_init.py

```diff
@@ -62,7 +62,7 @@ def test_poolconnect(self):
                                       (permission denied with error -1001)
            Above 3 testcases are defined in the yaml file.
 
-        :avocado: tags=all,pr,medium,pool,security_basic,security
+        :avocado: tags=all,pr,full_regression,small,pool,sec_basic,security
```

Comment:
I think the idea is to either have a test in 'pr' group or 'full_regression' group. Not in both.

Reply:
Thank you for the review and comment. 
It was requested by DE and System test to add this testcase to regression, I would keep both tags.